templates/todo/projects/nodejs-mongo/.repo/bicep/infra/main.bicep

```diff
@@ -14,7 +14,7 @@ param principalId string = ''
 
 var resourceToken = toLower(uniqueString(subscription().id, name, location))
 var tags = { 'azd-env-name': name }
-var abbrs = loadJsonContent('../../../../common/infra/abbreviations.json')
+var abbrs = loadJsonContent('../../../../../../common/infra/bicep/abbreviations.json')
```

Comment:
I think, if we make this change to move to a .repo folder, then we'll also want to udpate all the other templates to match that structure.

Reply:
I will create an issue to track the update of the remaining templates since we agreed that it will be addressed later